config/all-in-one/templates/eventmanager-app.yaml

```diff
@@ -0,0 +1,89 @@
+{{- if .Values.cp4waiops.eventManager.enabled }}
```

Comment:
Suggest to rename for consistency as below:
- File name `eventmanager-app.yaml` to be `eventmanager.yaml`
- ApplicationSet `metadata.name` to be `cp4waiops-eventmanagers`
- Application `metadata.name` to be `{{name}}-cp4waiops-eventmanager`

Reply:
Done